Procfile.dev

```diff
@@ -1,4 +1,3 @@
 server: yarn start:dev-server
 type-check: cd client && tsc --noEmit --watch
 web: yarn start:client
-ssr: yarn watch:ssr
```

Comment:
Just saw this now, was this line removed intentionally?

Reply:
I checked, and this was removed as part of 27353f572848b8f6cc98be1ce67c1e88d30d43dd.

Reply:
Reverted via f1211d321.